### PR TITLE
Massive - Change Request Details, Tons of Interfaces and Example Data

### PR DESCRIPTION
**Pull request by jamescd18**
I got carried away and accidentally wrote a whole bunch of code. I originally intended to just work on the change request details page, but then realized I needed the interface for change requests. I got carried away with building / expanding a bunch of the interfaces and also the example data to accompany those interfaces. I also realized I needed a bunch of display helper functions so I added the pipes.tsx file. Some screenshots below for the change requests detail pages. (Requesting review from @lpfahler16 so you can take a look) ~~(I still need to write tests)~~

Standard Change Request:
Screen Shot 2021-03-06 at 10 14 51 PM

Activation Change Request:
Screen Shot 2021-03-06 at 10 15 04 PM

Stage Gate Change Request:
Screen Shot 2021-03-06 at 10 15 26 PM

### CODE REVIEW

File: src/shared/pipes.tsx
Comment (kevinyu328) [inline src/shared/pipes.tsx:1]: Love the pipes. Definitely makes the code cleaner with its abstraction.

**Review by kevinyu328 — APPROVED** [review]
Very clean and easy to read. Love the pipes and the delegation used to create the change request details. Should I wait until after the tests are written to merge?

**Comment by jamescd18** [comment]
Should be done and ready for review now. Sorry again for the massive size.

**Review by kevinyu328 — APPROVED** [review]
Looks good! Sorry for the super long delay

**Pull request merged**
